cmd/juju/commands/enableha.go

```diff
@@ -66,8 +66,9 @@ type enableHACommand struct {
 
 const enableHADoc = `
 To ensure availability of deployed applications, the Juju infrastructure
-must itself be highly available.  enable-ha must be called
-to ensure that the specified number of controllers are made available.
+must itself be highly available. The enable-ha command will ensure
+that the specified number of controller machines are used to make up the
+controller.
```

Comment:
... are used to make up the controller cluster.

Perhaps? TBH, "...the specified number of controllers...." reads ok to me.